aya/src/programs/sock_ops.rs

```diff
@@ -2,13 +2,27 @@
 use std::os::fd::AsFd;
 
 use crate::{
-    generated::{bpf_attach_type::BPF_CGROUP_SOCK_OPS, bpf_prog_type::BPF_PROG_TYPE_SOCK_OPS},
+    generated::{
+        bpf_attach_type::BPF_CGROUP_SOCK_OPS, bpf_prog_type::BPF_PROG_TYPE_SOCK_OPS,
+        BPF_F_ALLOW_OVERRIDE, BPF_F_ALLOW_MULTI,
+    },
     programs::{
         define_link_wrapper, load_program, ProgAttachLink, ProgAttachLinkId, ProgramData,
         ProgramError,
     },
 };
 
+bitflags::bitflags! {
+    /// Flags passed to [`SockOps::attach()`].
+    #[derive(Clone, Copy, Debug, Default)]
+    pub struct SockOpsFlags: u32 {
```

Comment:
These aren't just SockOps flags though.
See: https://elixir.bootlin.com/linux/latest/source/include/uapi/linux/bpf.h#L1153

This should probably live in `links.rs` and be called `CgroupAttachFlags`.

Reply:
Good call.  Done.